assets/components/bundle/bundle.jsx

```diff
@@ -3,7 +3,6 @@
 // ----- Imports ----- //
 
```

Comment:
A shared component is created here but without scss file. I think we need to extract the style of this component from the pages and create a bundle.scss in order to be consistent with the file structure.